deps/rabbitmq_management_agent/src/rabbit_mgmt_metrics_collector.erl

```diff
@@ -583,6 +583,20 @@ insert_entry_op(Table, Key, Entry, Ops) ->
                                      end, {insert_entry, Entry}, TableOps0),
     maps:put(Table, TableOps, Ops).
 
+maybe_insert_entry_ops(Name, Table, Id, Incr, Entry, Ops, Policies) ->
+    case needs_filtering_out(Name) of
+        true -> Ops;
+        false -> insert_entry_ops(Table, Id, Incr, Entry, Ops, Policies)
+    end.
+
+needs_filtering_out(#resource{name = Name}) ->
+    case rabbit_mgmt_agent_config:get_env(filter_aggregated_queue_metrics_pattern) of
```

Comment:
This will likely call `application:get_env/2` or similar, which is very expensive. We should cache this value.

Reply:
Good point, I'll rework this